drivers/ethernet/eth_nxp_s32_netc_psi.c

```diff
@@ -220,6 +220,10 @@ BUILD_ASSERT(DT_NUM_INST_STATUS_OKAY(nxp_s32_netc_psi) == 1, "Only one PSI enabl
 		.NumberOfTxBDR = 1,						\
 		.SIVlanControl = (NETC_F3_PSICFGR0_SIVC_CVLAN_BIT		\
 				| NETC_F3_PSICFGR0_SIVC_SVLAN_BIT),		\
+		.changeMACAllowed = true,					\
```

Comment:
if I understand correctly this is a bug not related to this pr scope that affects `main`. So please log the bug and open a separate pr to fix it.

Reply:
I loggeg a bug and open a PR https://github.com/zephyrproject-rtos/zephyr/pull/66199 to fix
But I temporarily keep this commit in until #66199 is merged because this PR also need the commit to test